install/debian-installation-guide.md

```diff
@@ -0,0 +1,47 @@
+# Install Kata Containers on Debian
+
+These install instructions are based on the Ubuntu packages, but we need to install an updated version of `librdb1`.
+Tested only on Debian 9 - Stretch (x86_64).
+
+1. Dependencies
+
+  We need a newer version of librdbd1 to satisfy requirements: https://packages.debian.org/sid/librbd1
+
+  add `unstable` repo to `/etc/apt/sources.list.d/unstable.list` sources list:
+  
+  ```
+  # for unstable packages
+  deb http://mirror.aarnet.edu.au/debian/ unstable main contrib non-free
+  deb-src http://mirror.aarnet.edu.au/debian/ unstable main contrib non-free
+  ```
+  
+  set it with a lower priority than stable, edit/create `/etc/apt/preferences.d/unstable`:
+  
+  ```
+  Package: *
+  Pin: release a=unstable
+  Pin-Priority: 10
+  ```
+
+  this ensures that apt doesn't try to pull unstable updates for all your packages!
```

Comment:
s/this/This/

Reply:
@zeigerpuppy , I solve this, the issue is that there is a typo, the package name is `sudo -E apt install -t unstable librbd1`, you have to change the `librdb1` to `librbd1`

Reply:
updated